include/EGL/eglplatform.h

```diff
@@ -73,14 +73,15 @@ typedef struct wl_display     *EGLNativeDisplayType;
 typedef struct wl_egl_pixmap  *EGLNativePixmapType;
 typedef struct wl_egl_window  *EGLNativeWindowType;
 
-#elif defined(__GBM__)
+#else
+
+struct gbm_device;
+struct gbm_surface;
```

Comment:
```suggestion
struct gbm_bo;
```
Last nit, then I can merge this PR

Reply:
updated